sklearn/src/cblas/cblas_sscal.c

```diff
@@ -0,0 +1,183 @@
+/* ---------------------------------------------------------------------
+ *
+ * -- Automatically Tuned Linear Algebra Software (ATLAS)
+ *    (C) Copyright 2000 All Rights Reserved
+ *
+ * -- ATLAS routine -- Version 3.2 -- December 25, 2000
+ *
+ * -- Suggestions,  comments,  bugs reports should be sent to the follo-
+ *    wing e-mail address: atlas@cs.utk.edu
+ *
+ * Author         : Antoine P. Petitet
+ * University of Tennessee - Innovative Computing Laboratory
+ * Knoxville TN, 37996-1301, USA.
+ *
+ * ---------------------------------------------------------------------
```

Comment:
I have modified `cblas_dscal.c` to obtain this one. I prefer to find the original version of that file but actually in ATLAS they use an older version of that file.

@fabianp Where have you found the `cblas_dscal.c` ?